packages/backend/package.json

```diff
@@ -66,6 +66,7 @@
     "graphql": "^16.6.0",
     "graphql-scalars": "^1.18.0",
     "graphql-tools": "^8.3.3",
+    "http-message-signatures": "link:../http-message-signatures",
```

Comment:
```suggestion
    "http-message-signatures": "workspace:../http-message-signatures",
```

Reply:
I think I led you astray.
It looks like this is the way to add it as a workspace:
```
pnpm --filter backend add -P http-message-signatures --workspace
```
which will also update `pnpm-lock.yaml` (which still has `link`) and result in:
```
    "http-message-signatures": "workspace:^0.1.2",
```
because `http-message-signatures`'s `package.json` has a `version`